includes/abstracts/class-wc-gateway-mondido-abstract.php

```diff
@@ -318,17 +113,12 @@ public function lookupTransactionByOrderId( $order_id ) {
 	 * @return array|\WP_Error
 	 */
 	public function captureTransaction( $transaction_id, $amount ) {
-		$result = wp_remote_get( 'https://api.mondido.com/v1/transactions/' . $transaction_id . '/capture', array(
-			'method'  => 'PUT',
-			'headers' => array(
-				'Authorization' => 'Basic ' . base64_encode( "{$this->merchant_id}:{$this->password}" )
-			),
-			'body'    => array(
-				'amount' => number_format( $amount, 2, '.', '' )
-			)
-		) );
+		$result = $this->transaction->capture($transaction_id, $amount);
+		if (is_wp_error($result)) {
+			return $result;
+		}
 
-		return $result;
+		return json_decode(json_encode($result), true);
```

Comment:
Why?

Reply:
the old api call decoded everything to an array, the new one is decoding to object, so to not break old code i just replaced the inner of the old functions with the new api and reencoded the result to array, to keep the old behaviour.
new code is using the transaction property directly